### DISCUSSION TITLE
Orka Desktop CLI

### DISCUSSION THREAD

**Discussion by kandji-danielchapa** [macstadium/orka-desktop#4] [Ideas]
Are there any plans to develop a command-line interface for interacting with VMs? Things like quick VM spin-up/spin-down, modification, and automation tasks would all benefit from a CLI. It wasn't super clear to me whether the `orka3` CLI can be used for management of these local VMs or not. Thanks in advance.

**Comment by jasoncdavis0** [comment]
Hi @kandji-danielchapa - We are working on CLI for engine underneath Orka Desktop. The ```orka3``` CLI is designed to interface with the Orka Cluster, as it takes care of orchestration of workloads between nodes. A new CLI will be introduced to manage the VMs on an individual Desktop node in the coming weeks.

**Reply by matheusvechietin-mb** [reply]
Hi @jasoncdavis0 hope you are fine, is there any update about Orka Desktop CLI? Thanks
